### PR TITLE
Upload code for hand gesture recognizer project

### PR DESCRIPTION
**Pull request by javiert01**
This is the code for the blog "How To Build a Virtual Camera with Hand Gesture Detection in JavaScript"

### CODE REVIEW

File: javascript/hand-gesture-recognizer/gestures.js [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+      ],
+      [
+        "setWeight",
+        "Index",
+        2
+      ]
+    ],
+    "enabled": true
+  }
+}
```
Comment (javoweb) [inline javascript/hand-gesture-recognizer/gestures.js:340]: Nitpick: Let's add an empty line at the end of the file.
```suggestion
}

```

File: javascript/hand-gesture-recognizer/gestures.js [hunk 2]
```diff
@@ -0,0 +1,340 @@
+const gestures = {
```
Comment (javoweb) [inline javascript/hand-gesture-recognizer/gestures.js:1]: What do you think we add a comment describing what is the purpose of this file and a reference to the documentation or instructions?

**Review by javoweb — COMMENTED** [review]
Hi, @javiert01, Thanks for working on this content. The code looks great. Just a suggestion. What do you think if we use images in English? Is there a reason to keep them in Spanish? Also, are the `.gif` files used in the code? I don't see any reference to those two files.

File: javascript/hand-gesture-recognizer/gestures.js (hunk 2 above)
Comment (javiert01) [reply javascript/hand-gesture-recognizer/gestures.js:1]: Sure! I have updated it with a small description

**Comment by javiert01** [comment]
Yeah, I have contacted with Daniel Pazmiño to update the assets in English. Thanks for the suggestion!

**Pull request merged**